metadata/cadastre/utah-parcels

```diff
@@ -0,0 +1,75 @@
+# Title
+
+Utah Basic Parcels
+
+## ID
+
+cbe4da2a-3edb-4a4a-b082-60f9371c8006
+
+## Brief Summary
+
+Database representing parcel boundaries for all 29 counties in Utah that includes basic information such as parcel address and generalized land ownership.
+
+## Summary
+
+The Basic Parcels dataset contains information such as the county name, parcel address, land owner type, and phone number for the county recorder. Landowners of privately owned parcels are not specified.
+
+## Description
+
+### What is the dataset?
+
+A parcel is a defined area of land that is owned by a public or private entity. The Utah Basic Parcels database includes a polygon layer of parcels for each of Utah's 29 counties. Each county provides its own data for UGRC, thus, users should expect some variation in schema and completeness between counties.
+
+### What is the purpose of the dataset?
+
+This database represents UGRC's ongoing efforts to compile statewide parcels following the passage of [HB113](https://le.utah.gov/~2005/bills/hbillenr/HB0113.htm) in 2005. Parcel data is used in a variety of contexts, including public safety, economic development, transportation, planning, and the provision of public services.
+
+### What does the dataset represent?
+
+Parcels in this database are represented as polygons. Parcel polygons contain attributes unique to each parcel such as a unique parcel number (PARCEL_ID), address for the parcel (PARCEL_ADD, PARCEL_CITY, PARCEL_ZIP), update information (ParcelsCur, ParcelsRec, ParcelsPub, ParcelYear, ParcelNotes), a generalized land ownership designation (OWN_TYPE), and the phone number for the county recorders office (RECORDER). Fields and field definitions may vary from county to county.
+
+### How was the dataset created?
+
+<!--- When talking about how the dataset was made, the original metadata talks about the "LIR Working Group", which I presume was the primary force behind the LIR parcels, but was that group also responsible for these basic parcels? Is the history of both datasets about the same? -->
+
+### How reliable and accurate is the dataset?
+
+This multi-purpose database is not a substitute for obtaining the most current, legal land records information from county authorities. These datasets do not represent exact legal boundaries of Utah Parcels and should be used for general analysis and visualization purposes only.
+
+## Credits
+
+### Data Source
+
+- UGRC
+- Counties
+
+### Host
+
+UGRC
+
+## Restrictions
+
+## License
+
+## Tags
+
+- Land ownership
+- Private land
+- Public land
+- HB113
+
+## Secondary Category
+
+Boundaries
+
+## Data Page Link
+
+[https://gis.utah.gov/products/sgid/cadastre/parcels/]
+
+## Update
+
+### Update Schedule
+
+UGRC updates the basic parcels for Davis, Salt Lake, Utah, Washington, and Weber counties, also known as the Big 5, every month. All other counties are on a rotating schedule ranging from quarterly to annually.
```

Comment:
You capitalized basic parcels everywhere else.
```suggestion
UGRC updates the Basic Parcels for Davis, Salt Lake, Utah, Washington, and Weber counties, also known as the Big 5, every month. All other counties are on a rotating schedule ranging from quarterly to annually.
```

Reply:
I approve this suggestion